apps/prairielearn/src/pages/instructorFileEditor/instructorFileEditor.ejs

```diff
@@ -5,8 +5,7 @@
     <%- include('../partials/head', {pageTitle: 'Edit ' + fileEdit.fileName}); %>
     <script src="<%= node_modules_asset_path('ace-builds/src-min-noconflict/ace.js') %>"></script>
     <script src="<%= node_modules_asset_path('crypto-js/crypto-js.js') %>"></script>
-    <script src="./file_edit?serve=client"></script>
-    <script src="<%= node_modules_asset_path('socket.io-client/dist/socket.io.min.js') %>"></script>
```

Comment:
AFAICT this was unused on the page and probably just blindly copy/pasted from another page, so I removed this.